### PR TITLE
[java] support --jwp and --w3c driver executable args in EdgeDriverService

### PR DESCRIPTION
**Pull request by nbquestnb**
- [ X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

---
This change is [Reviewable](https://reviewable.io/reviews/seleniumhq/selenium/6485)

### CODE REVIEW

**Comment by presidenten** [comment]
Can we get this PR up to date and working? Or is there some other way to add --jwp flag to MicrosoftWebDriver.exe?

**Comment by evnraja32** [comment]
Looks i'm also facing problem with this --jwp. 

## Please refer:
https://github.com/SeleniumHQ/selenium/issues/7500

File: java/client/src/org/openqa/selenium/edge/EdgeDriverService.java [hunk 1]
```diff
@@ -53,6 +53,19 @@
...
 
+  /**
+   * Boolean system property that defines whether the MictosoftWebDriver executable
+   * should use the a protocol dialect compliant with the W3C WebDriver Specification.
+   * <p>
+   * Setting this property to a non-<see langword="null"/> value for driver
+   * executables matched to versions of Windows before the 2018 Fall Creators
+   * Update will result in a the driver executable shutting down without
+   * execution, and all commands will fail. Do not set this property unless
+   * you are certain your version of the MicrosoftWebDriver.exe supports the
```
Comment (PAX523) [inline java/client/src/org/openqa/selenium/edge/EdgeDriverService.java]: JavaDoc broken?

**Comment by AutomatedTester** [comment]
thanks for your effort in this PR but it is no longer needed as all drivers are w3c compliant

**Pull request closed without merging**